js/Localization.js

```diff
@@ -0,0 +1,114 @@
+import i18n from 'i18n-js';
+import * as RNLocalize from 'react-native-localize';
+
+i18n.translations = {
+  en: {
+    add_first_site: '+ Add your first site',
+    add_site: 'Add Site',
+    add_sites: 'Add Discourse sites to keep track of.',
+    add: '+ Add',
+    all: 'All',
+    and: 'and',
```

Comment:
A conjunction is usually a sign that translated strings are concatenated. It might be better to use an explicit translation for it, e.g. `%{foo} and %{bar}`.